include/erlang_ls.hrl

```diff
@@ -469,6 +469,21 @@
                                , containerName => binary()
                                }.
 
+%%------------------------------------------------------------------------------
+%% Signatures
```

Comment:
This is actually independent from hovering. The reason from introducing this is that I started implementing the `signatureHelp` method, then realized the `hover` was already there (some overlap exists between the two methods). Since this could still turn useful in the future, leaving it for now.